ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/DropSourceCommand.java

```diff
@@ -20,9 +20,10 @@
 import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
 import io.confluent.ksql.metastore.MutableMetaStore;
 import io.confluent.ksql.metastore.model.StructuredDataSource;
+import io.confluent.ksql.metastore.model.StructuredDataSource.DataSourceType;
```

Comment:
How come in some files this is a static import and sometimes it's not?

Reply:
Depends if its imported because there are _instances_ of the enum or just the use of the _constants_.